lib/workers/repository/process/extract-update.spec.ts

```diff
@@ -18,7 +30,7 @@ jest.mock('../../../util/git');
 const branchify = mocked(_branchify);
 const repositoryCache = mocked(_repositoryCache);
```

Comment:
As a GitLab user, I'm really glad seeing that MR being finished. Looking at the changes, I see that "cache" keyword is only used in test. I wonder, if those calls will be automatically cached with configured Redis as the other repository call (to minimize the number of calls in subsequent executions)?

Reply:
The entire OSV database is going to be bundled with Renovate, as part of https://github.com/renovatebot/osv-offline. There won't be any HTTP calls made at all.

Reply:
Ok, I've missed it...

Reply:
Sorry to dig out this old thread, but playing with this feature, I wonder, if (I'm not a JS/TS developer) using `import { Ecosystem, Osv, OsvOffline } from '@renovatebot/osv-offline'` will use the latest version of that "module" or to have the latest "database", I need to constantly update the Renovate image itself?

Reply:
The databases are stored in GitHub releases[^1] and a new database is downloaded if the current one is more than 1 day old:

https://github.com/renovatebot/osv-offline/blob/6ed04c96cf7937b889f98d1ad065dc252acef561/packages/osv-offline/src/lib/download.ts#L30-L35

No need to update the `osv-offline` package.

[^1]: https://github.com/renovatebot/osv-offline/releases